ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -273,14 +266,6 @@ final class SandboxServer(
         "write" -> indexAndWriteService.writeService,
       )
       observingTimeServiceBackend = timeServiceBackendO.map(TimeServiceBackend.observing)
```

Comment:
I think this is unused now.

Reply:
Also, you can probably delete `TimeServiceBackend.observing` and any related code.

Reply:
Ooohhh, very good catch. Thank you!